### PR TITLE
Adding .editorconfig file and rules to it.

### PR DESCRIPTION
**Pull request by Rucha1499**
Adding .editorconfig file and rules to maintain consistent coding style across the repo

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
+tab_width = 2
+trim_trailing_whitespace = true
```
Comment (swarajpure) [inline .editorconfig:9]: I guess we should also add `insert_final_newline = true`
Why? I read up some stackoverflow answers: https://stackoverflow.com/a/37684763/11829874

and ESLint doc also states this rule: https://eslint.org/docs/rules/eol-last

What do you say?

Comment (swarajpure) [reply .editorconfig:9]: One more reference: https://thoughtbot.com/blog/no-newline-at-end-of-file

Comment (swarajpure) [reply .editorconfig:9]: My bad sorry. It's already there

**Review by swarajpure — APPROVED** [review]

**Review by whyDontI — APPROVED** [review]
👍

**Pull request merged**